stack.go

```diff
@@ -51,8 +51,17 @@ func (x *stackType) top() float64 {
 }
 
 // printTop displays the top of the stack using the base indicated.
-func (x *stackType) printTop(base int) {
-	color.Cyan("= %s", formatNumber(x.top(), base))
+func (x *stackType) printTop(base int, args ...bool) {
```

Comment:
Is this function working at all? What's the "args ...bool" for? I can't understand what is happening here. There's also a bare `println` that shouldn't work.
 
 If we want to avoid the '=' sign on single commands mode, one option is to pass single to this method and treat it differently. Another way is to not call printTop, instead just print the top of the stack directly with ` `fmt.Println(stack.top())` from outside the function.

Reply:
> Is this function working at all?

Yep, it's working just great as you can see in the "Exempli Gratia" in my OP :smiley: 

> What's the "args ...bool" for? I can't understand what is happening here.

The idea was to change the code by the minimum possible amount. So, I added the `args ...bool` part to  stack.go/printTop() to     turn it into a variadic function (one accepting a variable number of arguments), and now it can be either called with a 2nd argument to indicate whether to "pretty-print" (with the "> " prefix and colors) the result, or not -- and that way I can call it from main.c/calc(), while all the rest of the code can continue to call it in the exact previous way (which will be handled in the exact same behavior as before, ie, will result in pretty-printing). That way, no changes are needed in any other parts of the code. 

> There's also a bare println that shouldn't work.

But work it does :smile: Again see the "Exempli Gratia" in my OP.

> If we want to avoid the '=' sign on single commands mode, one option is to pass single to this method and treat it differently.

That's exactly what I did from main.go/calc(), the variadic change above is just to avoid changing any other calls elsewhere. But if you prefer it the KISS way, I can remove the variadic and just add a normal parameter, and then change the other calls to add that parameter -- but then I would have to pass `single` somehow on to them too, so not as KISS as you might have planned. 

> Another way is to not call printTop, instead just print the top of the stack directly with  fmt.Println(stack.top())` from outside the function.

I thought about that too, but it would mean 'intruding' into stack.go internals, so not as 'clean' as the other solutions pointed above. 

Please consider my points above and tell me how you prefer it, and I will get it  done :+1:

Reply:
Done as discussed in our chat, PR updated.